runtime/compiler/build/toolcfg/aix-xlc/common.mk

```diff
@@ -114,6 +112,20 @@ ifeq ($(BUILD_CONFIG),prod)
     CX_FLAGS+=$(CX_FLAGS_PROD)
 endif
 
+ifeq (,$(findstring xlclang,$(notdir $(CC))))
+  # xlc options
+  CX_FLAGS+=-qnotempinc -qmbcs
+else
+  # xlclang
+  CX_FLAGS+=-qxlcompatmacros
+  SPP_FLAGS+=-qlanglvl=extc99
```

Comment:
@vsebe do you have a record for why this option was needed?

Reply:
This is related to string concatenation - see also https://github.com/eclipse/openj9/pull/6480.
extc99 is one of the supported defaults for xlclang (as per https://www.ibm.com/support/knowledgecenter/SSGH3R_16.1.0/com.ibm.xlcpp161.aix.doc/compiler_ref/opt_langlvl_aix.html)

Reply:
The illegal code was fixed: do we still need `-qlanglvl=extc99`?

Reply:
yes, we do

Reply:
Please explain why we need that option. Where & how does it make a difference?

Reply:
It is require to preprocess ebb.spp, e.g. `xclang -qlanglvl=extc99 ... -E openj9/runtime/compiler/p/runtime/ebb.spp`

Reply:
It was required before #6480 was merged, but is it *still* required?

Reply:
Yes, it is still required.